tsconfig.build.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Konfig for typedeklarasjonsgenerering. Lager kun deklarasjonsfiler for src/bedriftsmeny